compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -1847,6 +1847,18 @@ declare_lint! {
     "codeblock attribute looks a lot like a known one"
 }
 
+declare_lint! {
+    /// The `invalid_rust_codeblock` lint detects Rust code blocks in
+    /// documentation examples that are invalid (e.g.: empty, not parsable as
```

Comment:
This is a little hard to parse for me.
```suggestion
    /// documentation examples that are invalid (e.g. empty, not parseable as
```